legal-api/src/legal_api/reports/report.py

```diff
@@ -1313,6 +1313,10 @@ class ReportMeta:  # pylint: disable=too-few-public-methods
             'filingDescription': 'Certificate of Name Change',
             'fileName': 'certificateOfNameChange'
         },
+        'certificateOfNameCorrection': {
+            'filingDescription': 'Certificate of Name Correction',
+            'fileName': 'certificateOfNameChange'
+        },
```

Comment:
Do you have a UI example of this?

Reply:
Does the list of outputs look any different in Filings UI?

Reply:
It is intentional that the fileName is the same as certificate of name change?

Reply:
filieName you see here is the filie name of the html template (using the same template) not the output file name

Reply:
![image](https://github.com/bcgov/lear/assets/64869009/fb6b9903-eaf7-44a9-8be5-f00b5fa65726)

Reply:
If you think the output name and link (in Filings UI) work fine, go ahead and resolve this conversation.